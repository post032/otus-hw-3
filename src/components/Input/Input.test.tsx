import React from "react";
import renderer from "react-test-renderer";

import { InputText } from "./InputText";
import { InputColor } from "./InputColor";

describe("Inputs", () => {
  it("renders InputText placeholder set to 'placeholder'", () => {
    expect(
      renderer.create(<InputText placeholder="placeholder" />).toJSON()
    ).toMatchSnapshot();
  });
  it("renders InputColor with default green value", () => {
    expect(
      renderer.create(<InputColor defaultValue="green" />).toJSON()
    ).toMatchSnapshot();
  });
});
