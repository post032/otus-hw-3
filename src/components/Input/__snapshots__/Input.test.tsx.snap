// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inputs renders InputColor with default green value 1`] = `
<input
  defaultValue="green"
  type="color"
/>
`;

exports[`Inputs renders InputText placeholder set to 'placeholder' 1`] = `
<input
  placeholder="placeholder"
  type="text"
/>
`;
